src/PopOver.tsx

```diff
@@ -58,16 +63,49 @@ function HotspotContent(props: HotspotContentProps) {
       );
     }
     case "Video":
-      //  "https://photo-sphere-viewer-data.netlify.app/assets/pictos/pin-red.png"; // changed to make linter happy until icons are ready
-      break;
+      return (
+        <ReactPlayer
+          url={props.hotspot.src}
+          controls={true}
+          style={{
+            maxWidth: "100%",
+            maxHeight: "70vh",
+          }}
+        />
+      );
+
     case "Audio":
-      break;
+      return (
+        <AudioPlayer
+          style={{ width: "50vh", maxHeight: "70vh" }}
+          showSkipControls={false}
+          showJumpControls={false}
+          showDownloadProgress={false}
+          src={props.hotspot.src}
+        />
+      );
     case "Doc":
-      break;
+      const docs = [{ uri: props.hotspot.content }];
+      return (
+        <DocViewer
+          style={{ width: "80vw", height: "70vh" }}
+          documents={docs}
+          pluginRenderers={DocViewerRenderers}
+        />
+      );
+
     case "PhotosphereLink":
       break;
     case "URL":
-      break;
+      return (
+        <Iframe
+          styles={{
+            width: "80vw",
+            height: "70vh",
+          }}
+          url={props.hotspot.src}
+        />
+      );
```

Comment:
I think it would be useful if the link was also displayed here so users could click it to open a new tab.

Is there are reason to use the react-iframe `<Iframe>` component instead of the native `<iframe>` HTML element?

Reply:
Sounds good, I added the link as the hotspot title if its a URL popover. If thats not what we want I will modify it. 
Also removed react-Iframe!